### PR TITLE
Alpine clearing for package identifier and creater

### PR DESCRIPTION
**Pull request by crvreddy**

### CODE REVIEW

File: src/LCT.PackageIdentifier/AlpineProcesser.cs [hunk 1]
```diff
@@ -0,0 +1,193 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2023 Siemens AG
```
Comment (karthika-g) [inline src/LCT.PackageIdentifier/AlpineProcesser.cs:2]: Change the year

File: src/LCT.PackageIdentifier/Model/AlpinePackage.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2023 Siemens AG
```
Comment (karthika-g) [inline src/LCT.PackageIdentifier/Model/AlpinePackage.cs:2]: Change the year

File: src/LCT.PackageIdentifier/AlpineProcesser.cs [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+using System.Linq;
+using System.Net;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace LCT.PackageIdentifier
+{
+    /// <summary>
+    /// The DebianProcessor class
```
Comment (karthika-g) [inline src/LCT.PackageIdentifier/AlpineProcesser.cs:26]: Change this to alpine

File: src/LCT.SW360PackageCreator/Model/AlpinePackage.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2023 Siemens AG
```
Comment (karthika-g) [inline src/LCT.SW360PackageCreator/Model/AlpinePackage.cs:2]: Correct the year

File: src/SW360IntegrationTest/Alpine/ComponentCreatorInitialAlpine.cs [hunk 5]
```diff
@@ -0,0 +1,168 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2023 Siemens AG
```
Comment (karthika-g) [inline src/SW360IntegrationTest/Alpine/ComponentCreatorInitialAlpine.cs:2]: change the year

File: src/SW360IntegrationTest/Alpine/PackageIdentifierInitialAlpine.cs [hunk 6]
```diff
@@ -0,0 +1,101 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2023 Siemens AG
```
Comment (karthika-g) [inline src/SW360IntegrationTest/Alpine/PackageIdentifierInitialAlpine.cs:2]: Change the year

File: src/TestUtilities/TestParamAlpine.cs [hunk 7]
```diff
@@ -0,0 +1,43 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2023 Siemens AG
```
Comment (karthika-g) [inline src/TestUtilities/TestParamAlpine.cs:2]: Change the year

File: src/LCT.Common/Constants/Dataconstant.cs [hunk 8]
```diff
@@ -23,6 +23,7 @@ public static class Dataconstant
         {"MAVEN", "pkg:maven"},
         {"PYTHON", "pkg:pypi"},
         {"CONAN", "pkg:conan"},
+        {"ALPINE", "pkg:apk/alpine"},
```
[the remark this answers is not in the document]
Comment (crvreddy) [reply src/LCT.Common/Constants/Dataconstant.cs:26]: Yes,Its official

File: src/LCT.PackageIdentifier.UTest/AlpineParserTests.cs [hunk 9]
```diff
@@ -0,0 +1,172 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2023 Siemens AG
+//
```
[the remark this answers is not in the document]
Comment (crvreddy) [reply src/LCT.PackageIdentifier.UTest/AlpineParserTests.cs:3]: Updated

File: src/LCT.PackageIdentifier/AlpineProcesser.cs (hunk 1 above)
Comment (crvreddy) [reply src/LCT.PackageIdentifier/AlpineProcesser.cs]: Updated

File: src/LCT.PackageIdentifier/Model/AlpinePackage.cs (hunk 2 above)
Comment (crvreddy) [reply src/LCT.PackageIdentifier/Model/AlpinePackage.cs:2]: Updated

File: src/LCT.PackageIdentifier/AlpineProcesser.cs (hunk 3 above)
Comment (crvreddy) [reply src/LCT.PackageIdentifier/AlpineProcesser.cs]: Updated

File: src/LCT.SW360PackageCreator/Model/AlpinePackage.cs (hunk 4 above)
Comment (crvreddy) [reply src/LCT.SW360PackageCreator/Model/AlpinePackage.cs:2]: Updated

File: src/SW360IntegrationTest/Alpine/ComponentCreatorInitialAlpine.cs (hunk 5 above)
Comment (crvreddy) [reply src/SW360IntegrationTest/Alpine/ComponentCreatorInitialAlpine.cs]: Updated

File: src/SW360IntegrationTest/Alpine/PackageIdentifierInitialAlpine.cs (hunk 6 above)
Comment (crvreddy) [reply src/SW360IntegrationTest/Alpine/PackageIdentifierInitialAlpine.cs]: Updated

File: src/TestUtilities/TestParamAlpine.cs (hunk 7 above)
Comment (crvreddy) [reply src/TestUtilities/TestParamAlpine.cs]: Updated

File: src/LCT.Common/CommonAppSettings.cs [hunk 10]
```diff
@@ -29,6 +29,7 @@ public class CommonAppSettings
         public static string SnapshotDownloadURL { get; set; } = $"https://snapshot.debian.org/archive/";
         public static string PyPiURL { get; set; } = $"https://pypi.org/pypi/";
```
Comment (karthika-g) [inline src/LCT.Common/CommonAppSettings.cs:30]: Update the version in appsettings and ca nuspec 6.0.3

**Review by sumanthkb44 — APPROVED** [review]
Approved

**Pull request merged**